### PR TITLE
LPS-195262 [Marketplace] Fixing the filter and the search results for the solutions and app page

### PR DESCRIPTION
**Pull request by ytuik**
https://liferay.atlassian.net/browse/LPS-195262

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl
Comment (jairmedeiros) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:1]: Essa abordagem esta funcionando mesmo?

**Comment by jairmedeiros** [comment]
@claricebrasil @ytuik, o .ftl do Solutions Search Results parece bastante diferente do App ainda.

**Comment by jairmedeiros** [comment]
@claricebrasil @welitonpe, vcs podem revisar essa PR pf? Se estive tudo ok mesmo, podem dar forward

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 1]
```diff
@@ -104,15 +105,15 @@
 </#if>
```
Comment (claricebrasil) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:105]: Hey, Bruno! You need to use the same approach that we used in the Apps Search Results component

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl
Comment (jairmedeiros) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:1]: problema de indentacao gostosinho

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Review by jairmedeiros — APPROVED** [review]

**Pull request closed without merging**